### PR TITLE
feat: groupby airtable import

### PR DESCRIPTION
**Pull request by dstala**
## Change Summary
Group by configuration import

## Change type
- [x] feat: (new feature for the user, not a new feature for build script)

## Test/ Verification
Provide summary of changes.

## Additional information / screenshots (optional)
Anything for maintainers to be made aware of

### CODE REVIEW

File: packages/nocodb/src/modules/jobs/jobs/at-import/at-import.processor.ts [hunk 1]
```diff
@@ -2120,6 +2128,84 @@ export class AtImportProcessor {
...
+        if (
+          datatype === UITypes.Date ||
+          datatype === UITypes.DateTime ||
+          datatype === UITypes.Links ||
+          datatype === UITypes.MultiSelect ||
+          datatype === UITypes.SingleSelect ||
+          datatype === UITypes.SingleLineText ||
+          datatype === UITypes.Formula ||
+          datatype === UITypes.Checkbox ||
+          datatype === UITypes.Collaborator
```
Comment (mertmit) [inline packages/nocodb/src/modules/jobs/jobs/at-import/at-import.processor.ts:2165]: missing UITypes.Number which we support through UI

File: packages/nocodb/src/modules/jobs/jobs/at-import/at-import.processor.ts [hunk 2]
```diff
@@ -2120,6 +2128,84 @@ export class AtImportProcessor {
...
+            direction: group.order,
+          });
+        } else {
+          // skip group by over other data types
+          updateMigrationSkipLog(
+            await sMap.getNcNameFromAtId(viewId),
+            colSchema.title,
+            colSchema.uidt,
+            `group config skipped; group over ${datatype}  not supported`,
+          );
```
Comment (mertmit) [inline packages/nocodb/src/modules/jobs/jobs/at-import/at-import.processor.ts:2178]: this is not directly related to this PR I just wanted to point out we keep those logs in memory while operating but never make use of them.
maybe we can log them with stats or maybe include in T.event

**Review by mertmit — APPROVED** [review]

File: packages/nocodb/src/modules/jobs/jobs/at-import/at-import.processor.ts (hunk 2 above)
Comment (dstala) [reply packages/nocodb/src/modules/jobs/jobs/at-import/at-import.processor.ts:2178]: will check in few instances & see how better we can log. will take it up in a separate PR
